db/seeds.rb

```diff
@@ -6,159 +6,54 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-require 'activerecord-import/base'
-require 'activerecord-import/active_record/adapters/postgresql_adapter'
-require 'csv'
-require 'json'
-require 'yaml'
+Dir[Rails.root.join('db', 'seeds', '*.seeds.rb')].each { |f| require f }
```

Comment:
Using a `*.seeds.rb` convention here in case any other `*.rb` files end up in `db/seeds`.